README.md

```diff
@@ -17,6 +17,20 @@ A highly opinionated WordPress Pro scaffold, built with ❤️ by [GRRR](https:/
 
 View the [prerequisites](https://github.com/grrr-amsterdam/wordpress-scaffold/wiki/Prerequisites) in the wiki for more details.
 
+## Run with Docker
```

Comment:
Although the section is small enough that it's fine to be in the README, I'm not quite sure it's more important than the Quick start section? Docker is just a small part of the setup, and not really about the WordPress scaffold itself. 

But maybe I'm making a too big of a deal about this. Maybe @HammenWS can chime in?

Plus:

- See also the updated README in #43 (should fit together as well). I think I'd still prefer the `Quick start` being higher, mentioning the docker section at the moment you need to do something with Docker during the `Quick start`.
- The Docker wiki page is not listed in the wiki index (aka `Home`). Let's add it at a proper place there as well. In `Getting started` would work for me!
- Reference the wiki from the [Prerequisites](https://github.com/grrr-amsterdam/wordpress-scaffold/wiki/Prerequisites) (I recalled adding a stub there).

Reply:
Maybe we could also add a small sentence to the `Requirements`, something like:

```
Unless you're running the included [Docker setup](Docker), the following requirement apply:
```

Reply:
I've removed everything from the README and added one line with a link to the Docker wiki page.